py3status/modules/bluetooth.py

```diff
@@ -29,18 +29,15 @@
 {'color': '#FF0000', 'full_text': u'BT'}
 """
 
-import dbus
+from pydbus import SystemBus
```

Comment:
This module ought to have dbus `Requires:` too just like kdeconnector, mpris, systemd, vpn_status, wwan.